app/controllers/graphs_controller.rb

```diff
@@ -0,0 +1,5 @@
+class GraphsController < ApplicationController
+  def index
+    @graphs = Graph.all.order(date: "ASC")
```

Comment:
今の時点では`@graphs`をすぐに対応しきれませんが，とりあえず関連付けだけ入れておきましょうか。

```suggestion
    @graphs = user.graphs.order(date: "ASC")
```